userspace/libsinsp/cri.cpp

```diff
@@ -19,21 +19,37 @@ limitations under the License.
 
 #include "cri.h"
 
+#include <chrono>
+#include <grpc++/grpc++.h>
+
 #include "sinsp.h"
 #include "sinsp_int.h"
 
+namespace {
+bool pod_uses_host_netns(const runtime::v1alpha2::PodSandboxStatusResponse& resp)
```

Comment:
is this common in the code base instead of static?

Reply:
I got tired of your (plural you) review comments asking me to switch from static to namespace{}. Make up your mind, will you? :P

Reply:
yeah i don't really care either way, i was just asking to ensure it's consistent